disnake/flags.py

```diff
@@ -55,6 +55,16 @@ def __init__(self, func: Callable[[Any], int]) -> None:
         self.__doc__ = func.__doc__
         self._parent: Type[T] = MISSING
 
+    def __eq__(self, other: Any) -> bool:
+        if isinstance(other, self.__class__):
```

Comment:
Since `alias_flag_value` is implemented as a subtype of `flag_value`, this results in `Intents.emojis != Intents.emojis_and_stickers`. (I know this isn't the point of this PR, but it is new behavior either way)
`__or__` avoids this by using `flag_value` instead of `self.__class__`.

Reply:
Makes sense. Fixed in 5e91bf8.